Resources/config/doctrine/BasePage.orm.xml

```diff
@@ -22,6 +22,7 @@
         <field name="templateCode" type="string" column="template" nullable="false"/>
         <field name="createdAt" type="datetime" column="created_at"/>
         <field name="updatedAt" type="datetime" column="updated_at"/>
+        <field name="targetRedirectType" type="integer" column="target_redirect_type" nullable="true" default="302"/>
```

Comment:
Yes :( Not sure how big of a deal this can be. I know that in our projects we have the schema update in our deployment process integrated and didn't have problems with it but this of course doesn't have to be true for every project out there.

Would be great to have that feature in there so I don't have to run X projects on a custom build.

I haven't worked with the event you mentioned, i'll need some more time to read into that.